packages/opentelemetry-cloud-monitoring-exporter/src/utils.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { TimeSeries } from './types';
+
+/** Returns an array with arrays of the given size. */
```

Comment:
I think this could be more clear, like maybe "Returns the given array partitioned into arrays of the given size".

Reply:
Updated to "Returns the minimum number of arrays of max size chunkSize, partitioned from the given array"